### DISCUSSION TITLE
Need help in setting up server on local with yarn

### DISCUSSION THREAD

**Discussion by iamnovaprince** [learningequality/kolibri-design-system#805] [Community]
I got following error after hitting **yarn install** 

`warning svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
warning vue-simple-markdown > highlightjs@9.16.2: Use the 'highlight.js' package instead https://npm.im/highlight.js
[2/4] Fetching packages...
error js-beautify@1.15.1: The engine "node" is incompatible with this module. Expected version ">=14". Got "10.24.1"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.`

My Node version is : 10.24.1

and when i upgrae my node version to 14.21.3 then i got following error 

`
warning svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
warning vue-simple-markdown > highlightjs@9.16.2: Use the 'highlight.js' package instead https://npm.im/highlight.js
[2/4] Fetching packages...
error kolibri-tools@0.15.4: The engine "node" is incompatible with this module. Expected version "10.x". Got "14.21.3"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
`

Please suggest some solution

**Comment by ozer550** [comment]
Hi @iamnovaprince! what yarn version are you using?

**Comment by iamnovaprince** [comment]
Hi @ozer550 , My Yarn version is 1.22.22

**Reply by ozer550** [reply]
Could you check your yarn.lock file and see if the required version of `js-beautify` is 1.6.14 or 1.6.12 and not 1.15.1? seems like there is something wrong with how yarn.lock should be getting in packages. did you make any changes or delete it?

**Reply by iamnovaprince** [reply]
![Annotation 2024-10-24 224311](https://github.com/user-attachments/assets/89bff9c0-e7c0-4f22-931a-9199c17b36ee)
As per yarn.lock, the js-beautify is 1.6.14 and 1.6.12 , and I didn't made any changes. Just forked the repo ,cloned it on my PC  and did `yarn install`

**Reply by iamnovaprince** [reply]
Update :
I just took a pull from the repo again and did `yarn install` with same configuration and server started successfully. Must have missed something earlier
Anyway, Thanks for support @ozer550 
I am closing the discussion
